Automattic/jetpack
# Blogging prompts: hide placeholder prompt by default

#### Changes proposed in this Pull Request:

Changes the default setting for the showing placeholder prompt when starting a new post to false.

#### Other information:


#### Jetpack product discussion

We've received feedback from multiple places the the placeholder prompt is confusing, so we're turning it off by default while we figure out next steps.

p1670877978371049-slack-C025Q5RK2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Delete the `jetpack_blogging_prompts_enabled` option on your site, if present.
* Enable [experimental blocks](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/extensions/README.md#experimental-extensions) on your site.
* Set the home page to show a list of posts.
* You should not see a placeholder prompt by default when starting a new post.
* Go to Settings > Writing and turn on the placeholder prompts.
* Now you should still see a placeholder prompt when starting a new post.
